grpc/types.go

```diff
@@ -97,6 +99,28 @@ func (m Method) ResponseName() string {
 
 var EmptyReq = Message{
 	Name: "EmptyReq",
+	Args: []Argument{
+		{
+			Name:    "opts",
+			Type:    TransactOptsReq.Name,
+			IsSlice: false,
+		},
+	},
+}
+
+var EmptyConstReq = Message{
+	Name: "EmptyConstReq",
+}
+
+var EmptyRsp = Message{
```

Comment:
So, it's not really empty, right ?
How about using `TransactionResp` or `TxResp`?

Reply:
OK. Rename to `Empty`.